src/sparseml/pytorch/yolov8/trainers.py

```diff
@@ -58,17 +114,117 @@ def train(self):
             self._do_train(int(os.getenv("RANK", -1)), world_size)
 
     def resume_training(self, ckpt):
-        return super().resume_training(ckpt)
+        # NOTE: this method is called at the end of super()._setup_train()
+        super().resume_training(ckpt)
+
+        if ckpt is not None:
+            # resume - set manager from checkpoint
+            if "recipe" not in ckpt:
```

Comment:
would this cause issues with nominal yolov5 flows?

Reply:
Good call out, I think we can just remove